fedora_messaging/twisted/protocol.py

```diff
@@ -288,13 +476,15 @@ def resumeProducing(self):
         """
         # Start consuming
         self._running = True
-        for queue_name in self._queues:
-            queue_object, _consumer_tag = yield self._channel.basic_consume(
-                queue=queue_name
+        for consumer in self._consumers.values():
+            queue_object, _ = yield consumer.channel.basic_consume(
+                queue=consumer.queue, consumer_tag=consumer.tag
+            )
+            deferred = self._read(queue_object, consumer)
+            deferred.addErrback(
+                lambda f: _log.failure, "_read failed on consumer {c}", c=consumer
```

Comment:
These 7 lines are duplicated in `consume()` but I'm not sure it's worthwhile to factor them out ;-)